src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,40 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer ferrari = new Manufacturer(null,"Ferrari", "Italy");
+        Manufacturer ford = new Manufacturer(null,"Ford", "USA");
+        Manufacturer toyota = new Manufacturer(null, "Toyota", "Germany");
```

Comment:
don't pass nulls into constructor it's a bad practice, use setters only, or create constructor with fewer number of parameters